lua/nvim-treesitter/parsers.lua

```diff
@@ -2101,6 +2102,48 @@ list.supercollider = {
   maintainers = { "@madskjeldgaard" },
 }
 
+list.superhtml = {
+  install_info = {
+    url = "https://github.com/kristoff-it/superhtml",
+    includes = { "tree-sitter-superhtml/src" },
+    files = {
+      "tree-sitter-superhtml/src/parser.c",
+      "tree-sitter-superhtml/src/scanner.c",
+    },
+    branch = "main",
+  },
+  filetype = "superhtml",
+  maintainers = { "@rockorager" },
+}
+
+list.supermd = {
+  install_info = {
+    url = "https://github.com/kristoff-it/supermd",
+    includes = { "tree-sitter/supermd/src" },
+    files = {
+      "tree-sitter/supermd/src/parser.c",
+      "tree-sitter/supermd/src/scanner.c",
+    },
+    branch = "main",
```

Comment:
```suggestion
```
You only need to specify branch names that are not the default.

Reply:
> You only need to specify branch names that are not the default.

I had originally tried this but get this error when installing:

```
nvim-treesitter[ziggy]: Failed to execute the following command:
{
  cmd = "mv",
  opts = {
    args = { "-f", "tree-sitter-ziggy-tmp/ziggy-master", "tree-sitter-ziggy" },
    cwd = "/home/tim/.local/share/nvim",
    stdio = {
      [2] = <userdata 1>,
      [3] = <userdata 2>
    }
  }
}
mv: cannot stat 'tree-sitter-ziggy-tmp/ziggy-master': No such file or directory
```

Specifying the branch resolves this. Any ideas?

Reply:
As I wrote above, you need to add lockfile entries. `branch` in this case means "just use HEAD YOLO", which clearly is not ideal ;)